### ISSUE TITLE
the example looks broken in README.md for Opal 0.8.0

### ISSUE THREAD

**Issue by rainchen** [opal/opal-rails#63]
FYI, I'm using rails 4.2.2.
I followed the instruction in README.md, using the example codes as:

```
# app/assets/javascripts/greeter.js.rb

puts "G'day world!" # check the console!

# Dom manipulation
require 'opal-jquery'

Document.ready? do
  Element.find('body > header').html = '<h1>Hi there!</h1>'
end

```

these will be output as:

```
/* Generated by Opal 0.8.0 */
Opal.modules["greeter"] = function(Opal) {
  Opal.dynamic_require_severity = "ignore";
  var $a, $b, TMP_1, self = Opal.top, $scope = Opal, nil = Opal.nil, $breaker = Opal.breaker, $slice = Opal.slice;

  Opal.add_stubs(['$puts', '$require', '$ready?', '$html=', '$find']);
  self.$puts("G'day world!");
  self.$require("opal-jquery");
  return ($a = ($b = $scope.get('Document'))['$ready?'], $a.$$p = (TMP_1 = function(){var self = TMP_1.$$s || this, $a, $b;

  return (($a = ["<h1>Hi there!</h1>"]), $b = $scope.get('Element').$find("body > header"), $b['$html='].apply($b, $a), $a[$a.length-1])}, TMP_1.$$s = self, TMP_1), $a).call($b);
};
```

and the html of layout is:

```
<!-- other scripts -->
<script src="/assets/application.self-a03a22d638ad9cc4cad6350ae4b348cb82212146c1ff8800e513ea41b4e80317.js?body=1" data-turbolinks-track="true"></script><script>
//<![CDATA[
      if (typeof(Opal) !== 'undefined') {
        Opal.mark_as_loaded("opal");
Opal.mark_as_loaded("application");
Opal.mark_as_loaded("corelib/runtime.self");
Opal.mark_as_loaded("jquery.self");
Opal.mark_as_loaded("jquery_ujs.self");
Opal.mark_as_loaded("turbolinks.self");
Opal.mark_as_loaded("rspec/matchers/built_in/have");
Opal.mark_as_loaded("diff/lcs");
Opal.mark_as_loaded("diff/lcs/hunk");
Opal.mark_as_loaded("fileutils");
Opal.mark_as_loaded("test/unit/assertions");
Opal.mark_as_loaded("coderay");
Opal.mark_as_loaded("optparse");
Opal.mark_as_loaded("shellwords");
Opal.mark_as_loaded("socket");
Opal.mark_as_loaded("uri");
Opal.mark_as_loaded("drb/drb");
        
      }

//]]>
</script>
  <meta name="csrf-param" content="authenticity_token" />
<meta name="csrf-token" content="VEr94Vmz3Grxv0qcmePwGRT2arfB8gQJWao3BfOusT5+U0EyziSOZjbuiLMG9YMOhwNSOaDG5h5aZ3T1T/X49g==" />
</head>
<body class="controller-colors">

<h1>Hello Opal</h1>
<p>This is simple Opal application</p>



</body>
</html>
```

But nothing goes to the Chrome console and nothing changed for the page.
It just looks like these codes do not get executed at all.
Any clue for this?

**Comment by rainchen** [comment]
I found the solution, I have to use `application.js.rb` but not `application.js`.
If this is a `MUST DO` config, it should be highlighted in the README.md.

**Comment by elia** [comment]
@rainchen yes, should also state that you can still use `application.js` but then you need to `load` the Opal modules (i.e. files) manually, e.g.:

```js
// application.js
//= require greeter
Opal.load('greeter')
```

would be great if you can send a PR for the README, if you can't no worries, just let me know :smile:

**Comment by rainchen** [comment]
@elia no problem, I will send a PR for this note

**Comment by elia** [comment]
Fixed by #68
